rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/core/JacksonCustomizations.java

```diff
@@ -175,6 +176,10 @@ public Sw360Module() {
             @JsonProperty("linkedReleases")
             abstract public Map<String, ProjectReleaseRelationship> getReleaseIdToUsage();
 
+            @Override
+            @JsonProperty("visibility")
+            abstract public Visibility getVisbility();
```

Comment:
typo getVisbility

Reply:
the typo is correct here, because the db field is named `visbility`......
So this code will override the db field `visbility` to `visiblity` by using the getter `getVisbility`.